pkg/image/registry/imagestreamimport/rest.go

```diff
@@ -305,13 +305,14 @@ func checkImportFailure(status api.ImageImportStatus, stream *api.ImageStream, t
 		if len(status.Tag) > 0 {
 			tag = status.Tag
 		} else if status.Image != nil {
-			if ref, err := api.ParseDockerImageReference(status.Image.DockerImageReference); err == nil {
+			ref, err := api.ParseDockerImageReference(status.Image.DockerImageReference)
+			if err == nil {
```

Comment:
There's no need for this change. We actually prefer the original way of handling errors.